docs/gambit/index.md

```diff
@@ -10,7 +10,7 @@ Mutation testing is a technique for evaluating and improving
 The key idea is to introduce faults, called mutants, to the program under test
```

Comment:
Is "faults" the right word? 
Is it more "an almost equivalent solidity contracts to the original contract with some flawed logic". Maybe rephrasing to something like: "The key idea is to introduce `mutants` which are contracts similar to the originals with some small changes that violates the correctness of the contract"

Reply:
The sentence was bad. We do not know for sure that the mutant violates the correctness of the contracts. I changed to these two sentences instead:
"Mutants are identical copies of the tested program with a single random change.
The objective is to assess how well a test suite can identify and detect these mutants."